packages/react/package.json

```diff
@@ -41,41 +41,39 @@
     "test": "jest --silent",
     "test:watch": "jest --watch"
   },
-  "devDependencies": {
+  "dependencies": {
     "@pixi/react-components": "*",
     "@pixi/react-fiber": "*",
     "@pixi/react-invariant": "*",
-    "@pixi/react-modular": "*",
+    "@pixi/react-modular": "*"
+  },
+  "devDependencies": {
+    "@pixi/app": "^7.0.0",
+    "@pixi/constants": "^7.0.0",
+    "@pixi/core": "^7.0.0",
+    "@pixi/display": "^7.0.0",
     "@pixi/events": "^7.1.1",
+    "@pixi/graphics": "^7.0.0",
+    "@pixi/math": "^7.0.0",
+    "@pixi/mesh-extras": "^7.0.0",
+    "@pixi/particle-container": "^7.0.0",
+    "@pixi/sprite": "^7.0.0",
+    "@pixi/sprite-animated": "^7.0.0",
+    "@pixi/sprite-tiling": "^7.0.0",
+    "@pixi/text": "^7.0.0",
+    "@pixi/text-bitmap": "^7.0.0",
+    "@pixi/ticker": "^7.0.0",
     "@testing-library/react": "^13.4.0",
-    "canvas": "^2.11.0",
     "jest-webgl-canvas-mock": "^0.2.3",
-    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-test-renderer": "^18.2.0"
   },
   "peerDependencies": {
-    "@pixi/math": ">=6.0.0",
-    "@pixi/app": ">=6.0.0",
-    "@pixi/ticker": ">=6.0.0",
-    "@pixi/display": ">=6.0.0",
-    "@pixi/core": ">=6.0.0",
-    "@pixi/sprite-animated": ">=6.0.0",
-    "@pixi/text-bitmap": ">=6.0.0",
-    "@pixi/graphics": ">=6.0.0",
-    "@pixi/mesh-extras": ">=6.0.0",
-    "@pixi/particle-container": ">=6.0.0",
-    "@pixi/constants": ">=6.0.0",
-    "@pixi/sprite": ">=6.0.0",
-    "@pixi/text": ">=6.0.0",
-    "@pixi/sprite-tiling": ">=6.0.0",
-    "prop-types": ">=15.0.0",
```

Comment:
I wasn't 100% about this, but I added these here because I believe that `peerDependencies`, unlike other deps, aren't transitory? That is npm won't install the `peerDependencies` of a project's dependency or devDependency. Seeing as the intention here is that most users will only install and consume the single `@pixi/react` package, I thought we'd want to make sure that these were also installed for them?

Reply:
I could be wrong about this but i believe the issue you ran into was that @pixi/react didn't define @pixi/react-components as a dependency because its getting bundled here

This meant that when you installed @pixi/react it didn't install @pixi/react-components and therefore didn't add the pixi peer dependencies

Reply:
I believe that's not the case: https://dev.to/arcanis/implicit-transitive-peer-dependencies-ed0